src/client/pages/signIn/components/Molecules/SignInInputs.jsx

```diff
@@ -0,0 +1,13 @@
+import { FormInput } from '#components/Atoms/FormInput/FormInput.jsx';
+import css from './SignInInputs.module.css';
+
+const SignInInputs = () => {
+  return (
+    <div className={css.SignInInputs}>
+      <FormInput type="email" inputName="email" placeholder="Email" darkOrLight="dark"/>
+      <FormInput type="password" inputName="password" placeholder="Password" darkOrLight="dark"/>
```

Comment:
When your components are doubling, or trippling some same properties its good idea while creating template of this component to make this property with some default value.

Reply:
Did you implement default prop @dag-szad ❓